tests/cli/test_load_with_config.md

```diff
@@ -0,0 +1,78 @@
+First make sure that there is no model loaded:
+
+```sh
+$ modelator reset
+...
+```
+
+Load a model and a configuration from a toml file:
+
+```sh
+$ modelator load model/Test1.tla --config model/Test1.config.toml
+...
+Loading OK ✅
+...
+```
+
+```sh
+$ modelator info
+Model:
+- constants: {}
+...
+- init: Init
+- model_path: model/Test1.tla
+- module_name: Test1
+- monitors: []
+- next: Next
+- operators: ['Init', 'InitB', 'Next', 'Inv', 'InvB']
+- variables: ['x']
+Config at model/Test1.config.toml:
+- config_file_path: None
+- constants: {}
+- examples: []
+- init: Init
+- invariants: ['Inv']
+- next: Next
+- params: {'cinit': None, 'config': None, 'length': 5, 'max_error': None, 'no_deadlock': True, 'view': None}
+- traces_dir: traces/Test1
+```
+
+```sh
+$ modelator typecheck
+Type checking OK ✅
+```
+
+Run `check` on the loaded model:
+
+```sh
+$ modelator check
+...
+- Inv OK ✅
+...
+```
+
+Run `check` on the loaded model overriding the property to check and passing some setting to the checker:
+
+```sh
+$ modelator check --invariants InvB --init=InitB --params length=3
```

Comment:
This does not seem to be working: even after passing the argument 3, I still get the length of a trace to be 5 (as default in the `.toml` file)

Reply:
The test is wrong. It should be `--length=3` instead of `--params length=3`.

Reply:
I checked again with the command `modelator check --invariants InvB --traces-dir=tracesF --init=InitB --length=3` (after pulling the latest commits), and the length of the counterexample is still 5. Could you double-check?